README.md

```diff
@@ -7,7 +7,32 @@
 
 Frontend Service Discovery on AWS is a solution to handle frontend releases on AWS using the [frontend discovery pattern](https://github.com/awslabs/frontend-discovery).
 
-The frontend discovery pattern improves the development experience when developing, testing, and delivering micro-frontends by making use of a shareable configuration describing the entry point of micro-frontends, as well as additional metadata that is used to deploy in every environment safely using Canary Releases.
+The **frontend discovery pattern** improves the development experience when developing, testing, and delivering micro-frontends by making use of a shareable configuration describing the entry point of micro-frontends, as well as additional metadata that is used to deploy in every environment safely using Canary Releases.
```

Comment:
Could improve the grammar where you say safely. Perhaps...

"as well as additional metadata that is used for safe deployments in each environment using Canary Releases"